src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/Learner.cs

```diff
@@ -0,0 +1,52 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Domain;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.ValueTypes
+{
+    public class Learner : ValueObject
+    {
+        public Learner(
+            Guid id, 
```

Comment:
Doesn't having identity mean this isn't a value object?

Reply:
I think this question is for Kevin, not sure why he did it as a ValueObject

Reply:
perhaps Id in this case is just a field and isn't used as Id? 😀 idk 🤷‍♂️

Reply:
Yep it started out as a simple object so probably should be an entity now.  I will refactor.